css/amp.css

```diff
@@ -473,7 +473,7 @@ i-amphtml-scroll-container.amp-active, i-amp-scroll-container.amp-active {
 }
```

Comment:
so this will get reverted? I don't fully understand the code here and the plan

Reply:
This addresses the problem noted here: https://github.com/ampproject/amphtml/issues/17824. 
> reset-on-refresh adds a loader hidden via visibility: hidden. The loading container is display: block !important and shoves the list items out of the amp-list.

I thought this was only an issue for `FLEX_ITEM` but it seems to apply to all layouts that use `refresh-on-reset`. I'm curious if we can remove the `!important` on the display block so that we can override it for 

```
amp-list > .i-amphtml-loading-container.amp-hidden {
  display: none;
}
```

Reply:
I've updated the code to what I'd like to propose. If this doesn't work, then I will alter the logic for `applyAutoResize` so that it is not applied on `reset-on-refresh` due to #17824.

Reply:
It would be safer to add !important to the display none case instead of removing it from amp-loader. It has higher specifically so should override the other important

Reply:
Yup! Made this change.